src/Public/Responses.ps1

```diff
@@ -1595,48 +1628,49 @@ function Send-PodeSignal {
         [switch]
         $IgnoreEvent
     )
-
-    # error if not configured
-    if (!$PodeContext.Server.Signals.Enabled) {
-        throw 'WebSockets have not been configured to send signal messages'
-    }
-
-    # do nothing if no value
-    if (($null -eq $Value) -or ([string]::IsNullOrEmpty($Value))) {
-        return
-    }
-
-    # jsonify the value
-    if ($Value -isnot [string]) {
-        if ($Depth -le 0) {
-            $Value = (ConvertTo-Json -InputObject $Value -Compress)
-        }
-        else {
-            $Value = (ConvertTo-Json -InputObject $Value -Depth $Depth -Compress)
+    process {
```

Comment:
Unless the functions being updated to have these begin/process blocks 100% needs them; which they shouldn't as they're not dealing with array values from the pipeline, I'd say to remove them. They cause a slight performance hit in PowerShell, as it's making PowerShell need to invoke extra scriptblocks.